notebooks/MAST/m_dwarf_flares_hst/m_dwarf_flares_hst.ipynb

```diff
@@ -0,0 +1,1412 @@
+{
```

Comment:
<p>Yeah, it's actually noon UTC because.... well I guess because "that's how people used to do it and we're not gonna change now". As Julia says, it's 00:00 Terrestrial Time.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>